src/stdatamodels/jwst/datamodels/schemas/multispec.schema.yaml

```diff
@@ -84,6 +84,26 @@ allOf:
             type: number
             fits_keyword: EXTR_Y
             fits_hdu: EXTRACT1D
+          xstart:
```

Comment:
In keeping with the datamodel names "extraction_x" and "extraction_y" (above), I suggest we make these names `extraction_xstart`, `extraction_xstop`, etc.